src/App.css

```diff
@@ -1,38 +1,80 @@
-.App {
-  text-align: center;
+body{
+margin: 0px;
+padding: 0px;
+overflow: hidden;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.main{
```

Comment:
instead of using main as a class please use a tag

Reply:
If we use div as a tag it will be applied to all.
So what should I use?

Reply:
use the <main> tag instead